.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: BUG
+labels: bug
```

Comment:
we found that it's useful to have the initial tag as "Bug: triage" or something, rather bug, and once the reported issue is triaged, a maintainer changes the label to "bug". Quite frequently reported "bugs" are not really bugs.

Reply:
Great suggestion! Will adjust...

Reply:
The link to the sklearn templates was pure gold. I copied those into my directory and adjusted just a few things that were sklearn specific to be Fairlearn-specific. Please take another look :-)